### PR TITLE
PAYARA-1896 Unified asadmin command to set health check configuration (Payara 4.x)

### PR DESCRIPTION
**Pull request by jbee**
P4 PR of #3663 and #3679

### CODE REVIEW

**Comment by jbee** [comment]
jenkins test please

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/SetHealthCheckConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        if (report.getExtraProperties() == null) {
+            report.setExtraProperties(new Properties());
+        }
+        return report;
+    }
+
+    private void updateConfig(HealthCheckServiceConfiguration config, AdminCommandContext context) {
+        final ActionReport report = initActionReport(context);
+        try {
+            ConfigSupport.apply(configProxy -> {
```
Comment (Pandrex247) [inline nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/SetHealthCheckConfiguration.java]: We can't use lambdas in parts of Payara 4, as we still support JDK7 here.

**Comment by jbee** [comment]
jenkins test please

**Comment by jbee** [comment]
jenkins test please

**Pull request merged**